apps/podcast/lib/helpers/platform_helpers.dart

```diff
@@ -0,0 +1,6 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+
+bool isWeb = kIsWeb || kIsWasm;
+bool isDesktop = !Platform.isAndroid && !Platform.isIOS;
```

Comment:
&& !isWeb?

Reply:
Hmmm, vet inte om jag anser att de behöver vara uteslutande.
isDesktop != isNativeDesktop tänker jag.

För oklart?

Reply:
isDesktop could absolutely be true when isWeb is true. That means you could use isWeb && isDesktop to know if you are running the app in web on a computer.

isWeb is mostly used to make sure we run the correct code path for things like setup for Firebase, login and such (where web and native diverges)

Reply:
Gucci!